### PR TITLE
Jupyter Notebook for Pandas session

### PR DESCRIPTION
**Pull request by Wences91**

### CODE REVIEW

File: src/02b-pandas.ipynb [hunk 1]
```diff
@@ -0,0 +1,1871 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdcb84da",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "    Firstly, you have to install the package using the Anaconda Prompt:\n",
+    "    <br>\n",
+    "    $ pip install pandas\n",
```
Comment (vtraag) [inline src/02b-pandas.ipynb]: Perhaps we could change this to `conda install pandas`? Or at least, explain to people that they can install things in two ways, either using `conda` or using `pip`. Some packages are more difficult to install using `pip` while installation is easier using `conda`.

File: src/02b-pandas.ipynb [hunk 2]
```diff
@@ -0,0 +1,1871 @@
...
+   "source": [
+    "df[['journal_title', 'pub_year']].drop_duplicates()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdf64077-cc6b-4ccc-9d7d-675fa7a50050",
+   "metadata": {},
+   "source": [
+    "As there are some NA values, the can be removed with the `dropna` method or using the `dropna` parameter in the `groupby` method. By default it removes all the NA cells in all the columns, but it is possible to specify one specific column(s) with the `subset` parameter."
```
Comment (vtraag) [inline src/02b-pandas.ipynb]: Small typo:
```suggestion
    "As there are some NA values, they can be removed with the `dropna` method or using the `dropna` parameter in the `groupby` method. By default it removes all the NA cells in all the columns, but it is possible to specify one specific column(s) with the `subset` parameter."
```

File: src/02b-pandas.ipynb [hunk 3]
```diff
@@ -0,0 +1,1871 @@
...
+   "source": [
+    "df[['n_cits', 'n_refs']].agg(['sum', 'min', 'max', 'mean', 'std'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39860993-f51c-4f50-b831-2623a6af8ad8",
+   "metadata": {},
+   "source": [
+    "It is also possible to apply specific opertations to each columns."
```
Comment (vtraag) [inline src/02b-pandas.ipynb]:
```suggestion
    "It is also possible to apply specific operations to each columns."
```

**Review by vtraag — COMMENTED** [review]
Great, thanks! I've spotted a few small typos, you can change them, or I can change them for you, what you prefer.

Regarding the installing of packages, I think it might be better to use `conda` instead of `pip`, or at least explain both of them? For some packages it is easier to install them using `conda` then using `pip`, especially on Windows.

File: src/02b-pandas.ipynb (hunk 1 above)
Comment (Wences91) [reply src/02b-pandas.ipynb]: Yes, it is perfect for me to include both options and explain `conda` before `pip`.

Comment (vtraag) [reply src/02b-pandas.ipynb]: OK, I've added a small explanation.

**Comment by vtraag** [comment]
OK, thanks, I've added a brief explanation of `conda` and `pip`. I'll merge the PR now.

**Pull request merged**